Purchases/Logging/Strings/PurchaseStrings.swift

```diff
@@ -163,6 +167,15 @@ extension PurchaseStrings: CustomStringConvertible {
                 "Verification error \(errorMessage)"
         case .begin_refund_request_unsupported:
             return "Tried to call beginRefundRequest in a platform that doesn't support it!"
+        case .begin_refund_for_entitlement_nil_customer_info(let entitlementID):
+            return "Failed to get entitlement \(entitlementID ?? "") for refund. " +
+                "CustomerInfo is nil."
+        case .begin_refund_no_entitlement_found(let entitlementID):
+            return "Could not find entitlement \(entitlementID ?? "") for refund."
+        case .begin_refund_no_active_entitlement:
+            return "Could not begin refund request. No active entitlement."
+        case .begin_refund_customer_info_error(let entitlementID):
+            return "Failed to get CustomerInfo to proceed with refund for entitlement \(entitlementID ?? "")."
```

Comment:
As a user I think it's going to be confusing to read `Failed to get CustomerInfo to proceed with refund for entitlement .`. Could this be refactored to make the entitlement not Optional or use a different message when it's `nil`?